realm/realm-library/src/objectServer/java/io/realm/SyncConfiguration.java

```diff
@@ -286,6 +304,13 @@ boolean isSyncConfiguration() {
     }
 
     /**
+     * @return {@code true} if this configuration is intended to open a synced Realm offline (example for client reset), {@code false} otherwise.
+     */
+    public boolean isOpenSyncedRealmOffline() {
```

Comment:
Since this is on a SyncConfiguration calling it `SyncedRealm` seems a bit like overkill, what about `isRealmOffline`, `isRealmForcedOffline` or perhaps just `isForcedOffline` ?